_scripts/pages/download.js

```diff
@@ -115,50 +116,44 @@ Promise.all([config, jQuery, Payment, modal]).then(([config, $, Payment]) => {
 
             // Free download
             if (Number.isNaN(paymentAmount) || paymentAmount < paymentMinimum) {
-                plausible('Payment: Skipped')
+                plausible('Payment', {
+                    meta: {
+                        Input: paymentAmount,
```

Comment:
I'm not sure we want Input here. We are going to end up with a list of mostly one events for every invalid value. I think `Amount` is good enough.

Reply:
I wanted to get the same info for all 3 payment results (Skip, Pay, Fail) to see what inputs were causing which action.

Reply:
Mhmm, alright. We will probably have a lot of noise though